README.md

```diff
@@ -47,17 +47,29 @@ PebblesDB requires `libsnappy` and `libtool`. To install on Linux, please use
 PebblesDB was built, compiled, and tested with g++-4.7, g++-4.9, and g++-5. It may not work with other versions of g++ and other C++ compilers. 
 
 ### Installation
-`$ cd pebblesdb/`  
-`$ autoreconf -i`  
-`$ ./configure`  
-`$ make`  
-`# make install`  
-`# ldconfig`  
+
+Using autotools:
+
+```shell
```

Comment:
not sure the shell line is required. Could you remove?

Reply:
done